tests/test_chat_page.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+import pytest
+from selenium import webdriver
+
+from constants.base import DRIVER_PATH, BASE_URL
+
+
+class TestChatMessage:
+    log = logging.getLogger("[ChatMessage]")
+
+    @pytest.fixture(scope="function")
+    def chat_page(self):
+        # Pre-conditions
+        driver = webdriver.Chrome(DRIVER_PATH)
+        driver.get(BASE_URL)
+        driver.implicitly_wait(1)
+        # Steps
+        yield TestChatMessage(driver)
```

Comment:
Don't know what you're trying to do, but this won't work. 
When you're getting the BASE_URL you're on StartPage, not ChatPage or any other page.
Chat by itself is not the Page it's an element like Header, so it should be implemented in same way.